rails5-deps/2.3.6/Dockerfile

```diff
@@ -1,40 +0,0 @@
-FROM socrata/ruby:2.3.6
```

Comment:
Hey, I see these files accidentally ended up multiple PRs. Are they now deleted locally so that doesn't keep happening?